life_expectancy/cleaning.py

```diff
@@ -37,41 +26,36 @@ def clean_data(df: pd.DataFrame, country: str) -> pd.DataFrame:
     df_eu_life_expectancy = df_eu_life_expectancy.melt(id_vars=["unit", "sex", "age", "region"], var_name="year")
 
     # ensure data types
-    df_eu_life_expectancy["year"] = df_eu_life_expectancy["year"].astype(int)
+    df_eu_life_expectancy["year"] = df_eu_life_expectancy["year"].astype(np.int64)
     df_eu_life_expectancy["value"] = df_eu_life_expectancy["value"].str.extract(r"(\d+\.\d)")
     df_eu_life_expectancy["value"] = df_eu_life_expectancy["value"].astype(float)
 
     # drop nulls
-    df_eu_life_expectancy = df_eu_life_expectancy.dropna()
+    df_clean_data = df_eu_life_expectancy.dropna().copy()
```

Comment:
**Question**: Why include the copy command here and not, say, at the beginning of the function?

Reply:
No specific reason, I think I just did not think about it, thanks for pointing it out :)